actionpack/lib/action_dispatch/routing/route_set.rb

```diff
@@ -95,9 +96,23 @@ def clear!
             @url_helpers_module.send  :undef_method, helper
           end
 
+          @custom_helpers.each do |helper|
+            path_name = :"#{helper}_path"
+            url_name = :"#{helper}_url"
+
+            if @path_helpers_module.method_defined?(path_name)
+              @path_helpers_module.send :undef_method, path_name
```

Comment:
I'm going to assume that `undef_method` was used because `remove_possible_method` uses that internally for the reason outlined in 0244c0d8f339385a9b420a0565b17d327bf25b13